tasks/db_deploy/db_deploy.py

```diff
@@ -191,7 +191,7 @@ def create_tables():
             DatabaseError: An error occurred.
     """
     ddl_dir = os.environ["DDL_DIR"]
-    table_dir = f"{ddl_dir}{SEP}tables"
+    table_dir = f"{ddl_dir}tables"
```

Comment:
ddl_dir already ends with '\'. Other code requires it to. Double slashes are probably ignored by whatever was handling this value.